tasks/49_cat-and-dog-years/cat-and-dog-years.js

```diff
@@ -0,0 +1,15 @@
+function humanYearsCatYearsDogYears(humanYears) {
+  let catYears = 15;
+  let dogYears = 15;
+  if (humanYears >= 2) {
+    catYears += 9;
+    dogYears += 9;
+  }
+  if (humanYears > 2) {
```

Comment:
This if block looks similar to the previous one, I think you need to combine it somehow